router-configuration/to_push/br-mdf-01

```diff
@@ -0,0 +1,701 @@
+                    eui-64;
+                }
+            }
+        }
+        unit 101 {
+            description hiSCALE-FAST;
+            family inet {
+                address 10.0.136.1/21;
+            }
+            family inet6 {
+                address 2001:470:f0fb:101::1/64;
+                address 2001:470:f0fb:101::/64 {
+                    eui-64;
+                }
+            }
+        }
+        unit 102 {
+            description hiSpeaker;
+            family inet {
+                address 10.0.2.1/24;
+            }
+            family inet6 {
+                address 2001:470:f0fb:102::1/64;
+                address 2001:470:f0fb:102::/64 {
+                    eui-64;
+                }
+            }
+        }
+        unit 103 {
+            family inet {
+                address 10.0.3.2/24;
+            }
+            family inet6 {
+                address 2001:470:f0fb:103::2/64;
+            }
+        }
+        unit 105 {
+            description hiAVLAN;
+            family inet {
+                address 10.0.5.1/24;
+            }
+            family inet6 {
+                address 2001:470:f0fb:105::1/64;
+                address 2001:470:f0fb:105::/64 {
+                    eui-64;
+                }
+            }
+        }
+        unit 107 {
+            description hiSigns;
+            family inet6 {
+                address 2001:470:f0fb:107::1/64;
+                address 2001:470:f0fb:107::/64 {
+                    eui-64;
+                }
+            }
+        }
+        unit 108 {
+            description hiStaff;
+            family inet {
+                address 10.0.8.1/24;
+            }
+            family inet6 {
+                address 2001:470:f0fb:108::1/64;
+                address 2001:470:f0fb:108::/64 {
+                    eui-64;
+                }
+            }
+        }
+        unit 110 {
+            description hiRegistration;
+            family inet {
+                address 10.0.10.1/24;
+            }
+            family inet6 {
+                address 2001:470:f0fb:110::1/64;
+                address 2001:470:f0fb:110::/64 {
+                    eui-64;
+                }
+            }
+        }
+        inactive: unit 200 {
+            description "VLANs 200-398 are on Expo switches and not directly routed here.";
+        }
+        unit 499 {
+            description "vendor_backbone -- Vender VLANS 200-398 are routed to switches via this network and OSPF";
+            family inet {
+                address 10.1.0.1/24;
+            }
+        }
+        unit 900 {
+            description "IP Link between buildings";
+            family inet {
+                address 172.20.0.1/24;
+            }
+            family inet6 {
+                address 2001:470:f0fb:8000::1/64;
+            }
+        }
+        unit 999 {
+            description "Internet Link to Hilton/AT&T";
+            family inet {
+                address 12.203.68.60/23;
```

Comment:
new IP for the hilton, looks like it matches your notes on the mailing list

Reply:
Correct.